### PR TITLE
FEATURE: Prometheus for each group

### LINKED ISSUE #36: Prometheus storage each group

### PR DESCRIPTION
**Pull request by iTitus**
Closes #36

### CODE REVIEW

File: src/main/java/mops/businesslogic/DirectoryService.java [hunk 1]
```diff
@@ -89,4 +89,19 @@ DirectoryPermissions updatePermission(Account account,
...
+     * @return directory count
+     */
+    long getDirCount(long groupId) throws MopsException;
+
+    /**
+     * Get the total number of directories in all groups.
+     *
+     * @return directory count
+     */
+    long getDirCount() throws MopsException;
```
Comment (Segelzwerg) [inline src/main/java/mops/businesslogic/DirectoryService.java]: This is a bit confusing, maybe rename one of them

File: src/main/java/mops/businesslogic/prometheus/PrometheusComponent.java [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+    @SuppressWarnings("PMD.LawOfDemeter") //This is a builder
+    private void addTotalGauges() {
+        log.debug("Adding total storage gauge.");
+        Gauge
+                .builder("mops.material1.totalStorageUsage", () -> {
+                            try {
+                                return fileInfoService.getStorageUsage();
+                            } catch (MopsException ignored) {
+                                return 0L;
```
Comment (Segelzwerg) [inline src/main/java/mops/businesslogic/prometheus/PrometheusComponent.java]: Shouldn't we log this? I'm mean for the stacktrace?

Comment (iTitus) [reply src/main/java/mops/businesslogic/prometheus/PrometheusComponent.java]: ah yeah
there is log.cathing(e) I think

File: src/main/java/mops/businesslogic/DirectoryService.java (hunk 1 above)
Comment (iTitus) [reply src/main/java/mops/businesslogic/DirectoryService.java]: sure

**Pull request merged**